go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/heketi/heketi
```

Comment:
This has to include the version:
```
module github.com/heketi/heketi/v10
```

Then all imports in your code also need to use that instead of plain `github.com/heketi/heketi`.

It works as-is when working inside the repo, but any consumers referencing the code run into https://github.com/heketi/heketi/issues/1897